integtest/spec/spec_helper.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require_relative 'helper/convert'
+require_relative 'helper/dsl'
+require_relative 'helper/sh'
+
+require 'tmpdir'
+require 'fileutils'
+require 'open3'
+
+ENV['GIT_AUTHOR_NAME'] = 'Test'
+ENV['GIT_AUTHOR_EMAIL'] = 'test@example.com'
+ENV['GIT_COMMITTER_NAME'] = 'Test'
+ENV['GIT_COMMITTER_EMAIL'] = 'test@example.com'
+
+RSpec.configure do |config|
+  # Enable flags like --only-failures and --next-failure
+  config.example_status_persistence_file_path = '.rspec_status'
+
+  # Disable RSpec exposing methods globally on `Module` and `main`
+  config.disable_monkey_patching!
```

Comment:
I picked this up from somewhere saying it was best practice not to use the monkey patched stuff and have brought it along every time I use rspec. It has been a while since I tracked down where the advice came from, but it seems to have served me well.